Tests.Integration.Presentation.Web/Advice/AdviceTest.cs

```diff
@@ -49,19 +49,19 @@ public async Task Can_Add_Advice_With_Multiple_Email_Receievers()
             //Arrange
             var recipient1 = new Core.DomainModel.Advice.AdviceUserRelation
             {
-                Name = $"{A<string>()}@test.dk",
+                Name = createWellformedEmail(),
```

Comment:
Still not DRY ..: private void SetupAdviceUserRelation(string name) { return new AdviceUserRelation { ... }; }